VERSION

```diff
@@ -1 +1 @@
-1.13.0
+1.14.0
```

Comment:
When merging to master, should `master` branch to be `1.14.0` or `1.15.0-dev`?

Reply:
It was 1.13 before this release.

Reply:
https://github.com/cortexproject/cortex/pull/4793/files

Reply:
Let's merge on green.